samples/benchmarks/coremark/README.rst

```diff
@@ -171,18 +167,33 @@ CONFIG_APP_MODE_FLASH_AND_RUN - Start CoreMark sample automatically after flashi
    Otherwise, it will wait for the button press.
 
 .. note::
-   The :kconfig:option:`CONFIG_APP_MODE_FLASH_AND_RUN` Kconfig option is always enabled for the PPR core.
-   This core does not use buttons.
+   The :kconfig:option:`CONFIG_APP_MODE_FLASH_AND_RUN` Kconfig option is always enabled for the PPR core on the ``nrf54h20dk/nrf54h20/cpuapp`` board target.
+   This core on the ``nrf54h20dk/nrf54h20/cpuapp`` board target does not use the on-board buttons and LEDs.
+
+.. _SB_CONFIG_APP_CPUFLPR_RUN:
```

Comment:
don't like this being a global doc tag because it's not generic enough to not clash with other things

Reply:
The `_APP_` infix suggests that this belongs to certain projects. This is the way in which we document Kconfig options at the CMake project level both for sysbuild and the main image. @peknis, @divipillai or @annwoj any ideas on how to move forward?

Reply:
This style is also used in other samples. For example:

https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/suit/flash_companion/README.html

Reply:
Since this is a sample specific Kconfig option, the Kconfig is defined in this manner - https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/templates/sample_README.html#configuration_options. 

The same is followed in all the other samples.

Reply:
But if I created another sample application and used the same Kconfig then documented it with the same tag e.g. `.. _SB_CONFIG_APP_CPUFLPR_RUN:` surely doc build is going to fail because of multiple definitions of the same thing?

Reply:
If that's the case and the same Kconfig names in two different samples cause documentation build failure, then we may need to devise a plan to handle this scenario at some point. CC: @divipillai, @annwoj, @peknis

Reply:
@nordicjm : Since it is defined in the Kconfig file of this sample - samples/benchmarks/coremark/Kconfig.sysbuild, the Kconfig is defined in this sample documentation using the "_SB_CONFIG_APP_CPUFLPR_RUN:" label. 

From the other sample docs, if the same Kconfig is used, we would just reference to this section using the :ref: syntax. For example, :ref:`SB_CONFIG_APP_CPUFLPR_RUN <SB_CONFIG_APP_CPUFLPR_RUN>`. We don't define the Kconfig again in another sample documentation.